.github/workflows/release.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       with:
         path: |
           .build
-          SourcePackagesCache
+          ../SourcePackagesCache
```

Comment:
as I see SourcePackagesCache is placed in project root folder, why `../` was added?